addon/globalPlugins/remoteClient/server.py

```diff
@@ -135,7 +144,7 @@ def do_join(self, obj):
 		self.authenticated = True
 		clients = []
 		client_ids = []
-		for c in self.server.clients.values():
+		for c in list(self.server.clients.values()):

```

Comment:
I don't think this is necessary

Reply:
You are probably right, but it also isn't a hot path so it isn't likely to hurt and maintains the original intent